.github/workflows/binaries.yaml

```diff
@@ -20,6 +22,7 @@ jobs:
       with:
         # Also ensure we have all history with all tags
         fetch-depth: 0
+        fetch-tags: true
```

Comment:
Maybe a comment that this term doesn't seem to help.

Reply:
Ah.. I should remove it.